apps/web/src/server/applications/case/representations/representation/check-answers/check-answers.middleware.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {import("express").Request} req
+ * @param {import("express").Response} res
+ * @param {import("express").NextFunction} next
+ */
+export const addRepresentationValuesToBody = async (req, res, next) => {
+	const { originalRepresentation, received, type, represented, representative } =
+		res.locals.representation;
+
+	req.body = {
```

Comment:
they are not from the form submission not sure why this would be put on a body ? only to use the express validator ?

Reply:
Correct, so we can generate the same error objects as we would on a form submission.